lutris/runners/tic80.py

```diff
@@ -0,0 +1,53 @@
+import os
+from lutris.runners.runner import Runner
+
+
+class tic80(Runner):
+    human_name = "TIC-80"
+    description = "TIC-80 tiny computer"
+    platforms = ['TIC-80']
+    runner_executable = 'tic80/tic80'
+    game_options = [
+    	{
+            'option': 'main_file',
+            'type': 'file',
+            'label': 'ROM file',
+            'default_path': 'game_path'
+        }
+    ]
+
+    runner_options = [
+        {
+            'option': 'surf',
+            'type': 'bool',
+            'label': 'Start in Surf',
+            'default': False
+        },
+        {
+            'option': 'skip',
+            'type': 'bool',
+            'label': 'Skip startup animation',
+            'default': False
+        },
+        {
+            'option': 'sound',
+            'type': 'bool',
+            'label': 'Start in silent mode',
+            'default': False
+        }
+    ]
+
+    def play(self):
+        """Run the game."""
+        arguments = [self.get_executable()]
+        if self.runner_config.get('surf'):
```

Comment:
I see a `-fullscreen` option we could use here too! https://github.com/nesbox/TIC-80/wiki#command-line-arguments

Reply:
Fullscreen doesn't seem to work. I created a issue on their [github](https://github.com/nesbox/TIC-80/issues/626), will add the option as soon as it works